core/common/junit/src/main/java/org/eclipse/edc/junit/matchers/EventEnvelopeMatcher.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
License header should be BMW

Reply:
Yeah it's still on draft, i will need to fix the dispatcher accordingly with the event hierarchy and i will then make it ready for review